distributed/scheduler.py

```diff
@@ -3860,9 +3863,12 @@ async def post(self):
             pc = PeriodicCallback(self.check_worker_ttl, self.worker_ttl * 1000)
             self.periodic_callbacks["worker-ttl"] = pc
 
-        pc = PeriodicCallback(self.check_idle, (self.idle_timeout or 1) * 1000 / 4)
+        pc = PeriodicCallback(self.check_idle, 250)
```

Comment:
I would be annoyed if I had set a timeout of 2h and the shutdown was 30min later than expected.
These methods are tiny and running them 4 times per second is inconsequential.